.github/workflows/.windows.yml

```diff
@@ -249,7 +174,6 @@ jobs:
     runs-on: ${{ inputs.os }}
     timeout-minutes: 120
     needs:
-      - build
```

Comment:
Related to previous comment, this is racy as we could call this reusable workflow without having a dependency on `build` so I think it's best to keep it `build` here.